lib/looker-sdk/client.rb

```diff
@@ -396,14 +397,16 @@ def build_response(connection, request)
           http.read_timeout = connection.options.timeout rescue 60
 
           http.request(http_request) do |response|
+            if response.code != "200"
+              error = LookerSDK::Error.from_response(response)
```

Comment:
this bypasses the network read error?

Reply:
The original code looked for 200 and did something, but had no handling for a non-200 response. This raises an error on any non-200.

Reply:
I handled the network read error that @aschott-looker was having in https://github.com/looker-open-source/looker-sdk-ruby/pull/100 using the same code that he monkey patched.